
# HG changeset patch
# User Hans de Goede <hdegoede@redhat.com>
# Date 1263748749 -3600
# Node ID 9977416a72e13998d36b305ebfd2cc5d0cc57825
# Parent  34a507499c368f97051ebabc21b759e88815e501
libv4l: Start of 0.6.5 dev cycle

From: Hans de Goede <hdegoede@redhat.com>

libv4l: Start of 0.6.5 dev cycle

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/ChangeLog	Sun Jan 17 18:08:01 2010 +0100
+++ b/v4l2-apps/libv4l/ChangeLog	Sun Jan 17 18:19:09 2010 +0100
@@ -1,3 +1,7 @@
+libv4l-0.6.5
+------------
+* Add more laptop models to the upside down devices table
+
 libv4l-0.6.4
 ------------
 * Add more laptop models to the upside down devices table
--- a/v4l2-apps/libv4l/Makefile	Sun Jan 17 18:08:01 2010 +0100
+++ b/v4l2-apps/libv4l/Makefile	Sun Jan 17 18:19:09 2010 +0100
@@ -1,5 +1,5 @@
 LIB_RELEASE=0
-V4L2_LIB_VERSION=$(LIB_RELEASE).6.4
+V4L2_LIB_VERSION=$(LIB_RELEASE).6.5-test
 
 all install:
 	$(MAKE) -C libv4lconvert V4L2_LIB_VERSION=$(V4L2_LIB_VERSION) $@

